webapp/src/widgets/tts/elevenlabs-ws-api.ts

```diff
@@ -36,24 +38,26 @@ export class ElevenlabsApi {
                 // const audioChunk = atob(response.audio);  // decode base64
                 console.log("Received audio chunk");
 
-                if (!this.audioContext) {
-                    await this.initAudioContext();
-                }
+                ttsPlayer.addArrayBuffer(decode(response.audio));
```

Comment:
Does the player automatically detect the availability of new buffer, and resume playing, accordingly?